src/command_modules/azure-cli-role/azure/cli/command_modules/role/tests/test_role.py

```diff
@@ -55,38 +52,45 @@ def tear_down(self):
         super().tear_down()
 
     def test_role_assignment_scenario(self):
-
         if self.playback:
-            #TODO: we should auto record generated guids and use them under playback.
-            #For now, get the values from the .yaml file and put in here.
-            with mock.patch('uuid.uuid4') as m:
-                m.side_effect = ['1518cef5-a2c8-48e1-acb5-eaadf2474f79', 'f13b65c4-7c62-4dd2-b2eb-2a348b57c25b']
-                self.execute()
+            return #live-only test, so far unable to replace guid in binary encoded body
```

Comment:
>return #live-only test, so far unable to re [](start = 12, length = 43)

If a solution isn't found before this is merged, we should create an issue (Pivotal since its internal?)